src/components/App.jsx

```diff
@@ -1,34 +1,154 @@
-import React, { useState } from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
+import Quote from '../routerpages/QuotePage';
+import Calculator from '../routerpages/CalculatorPage';
 
-import Header from './Header';
-import Display from './Display';
-import ButtonContainer from './ButtonContainer';
-import calculate from '../logic/calculate';
+const DefaultPageContent = () => (
+  <div className="font-semibold">
+    <p className="text-3xl">Welcome to our page!</p>
+    <p className="my-10">
+      Hey there. I&apos;m Ibrohim. I&apos;m a software developer. I can help you build a product,
+      feature, or website. Look through some of my work and experience! If you like what you see and
+      have a project you need coded, don&apos;t hesitate to contact me.
+    </p>
+    <p className="my-10">
+      calculator app, designed to simplify your mathematical tasks effortlessly. With an intuitive
+      interface and advanced functionality. click here to visit{' '}
+      <Link className="text-blue-300 underline" to="/calculator">
+        Calculator{' '}
+      </Link>
+    </p>
+  </div>
+);
```

Comment:
Can we move this to the `pages` folder? It can be a component to be imported over here.